ArduPlane/mode_autoland.cpp

```diff
@@ -237,5 +274,16 @@ void ModeAutoLand::arm_check(void)
     }
 }
 
+// climb before turn altitude check
+void ModeAutoLand::check_altitude(void)
+{
+    const uint16_t altitude = plane.relative_ground_altitude(false,false);
```

Comment:
Sorry clicked the wrong button. this is also meters I think.

Reply:
ditto